### PR TITLE
Make Boost available for CUDA

### PR DESCRIPTION
**Pull request by bernhardmgruber**
Fixes: #2701

### CODE REVIEW

**Comment by partouf** [comment]
You also have to add "boost" to the libs= line

**Comment by bernhardmgruber** [comment]
> You also have to add "boost" to the libs= line

I did add `Boost`, but I did not comment on it. I fixed the spelling, thx!

File: etc/config/cuda.amazon.properties [hunk 1]
```diff
@@ -375,7 +375,48 @@ compiler.nvrtc120u1.exe=/opt/compiler-explorer/cuda/12.0.1/bin/nvrtc_cli
 compiler.nvrtc121.semver=12.1.0
 compiler.nvrtc121.exe=/opt/compiler-explorer/cuda/12.1.0/bin/nvrtc_cli
 
-libs=bmulti:cueigen:thrustcub:cucub:cudacxx:nvtx:nsimd:cuco:hip-amd
+libs=boost:bmulti:cueigen:thrustcub:cucub:cudacxx:nvtx:nsimd:cuco:hip-amd
```
Comment (bernhardmgruber) [inline etc/config/cuda.amazon.properties:378]: I added `boost` here.

**Review by jeremy-rifkin — APPROVED** [review]
Looks good to me, thanks for the pr

**Comment by mattgodbolt** [comment]
This is now live: thanks!

**Comment by bernhardmgruber** [comment]
Thank you for considering this change! :)

**Pull request merged**